src/coreneuron/mechanism/register_mech.cpp

```diff
@@ -212,7 +213,7 @@ void hoc_register_dparam_semantics(int type, int ix, const char* name) {
        xx_ion and #xx_ion which will get
        a semantics value of -1, -2, -3,
        -4, -5, -6, -7, -8, -9, -10,
-       type, and type+1000 respectively
```

Comment:
I reviewed every occurrence of 1000 and am reasonably confident there are no cases remaining which use 1000 in this context.